esphome/components/output/__init__.py

```diff
@@ -58,6 +64,10 @@ async def setup_output_platform_(obj, config):
         cg.add(obj.set_min_power(config[CONF_MIN_POWER]))
     if CONF_ZERO_MEANS_ZERO in config:
         cg.add(obj.set_zero_means_zero(config[CONF_ZERO_MEANS_ZERO]))
+    if CONF_MIN_TURN_ON_POWER in config:
+        cg.add(obj.set_min_turn_on_power(config[CONF_MIN_TURN_ON_POWER]))
+    if CONF_MIN_TURN_ON_POWER_DELAY in config:
+        cg.add(obj.set_min_turn_on_power_delay(config[CONF_MIN_TURN_ON_POWER_DELAY]))
```

Comment:
Having defaults means they will always be in config
```suggestion
    cg.add(obj.set_min_turn_on_power(config[CONF_MIN_TURN_ON_POWER]))
    cg.add(obj.set_min_turn_on_power_delay(config[CONF_MIN_TURN_ON_POWER_DELAY]))
```

Reply:
Are you sure about this? When I commit your suggestion the tests fail with `KeyError: 'min_turn_on_power'`.